specification/templates/.github/CODEOWNERS

```
@@ -1,5 +1,4 @@
-# Learn about CODEOWNERS file format:
 #  https://help.github.com/en/articles/about-code-owners
 #
 # Global Owners
-* @signalfx/TEAM-maintainers
+* gdi-TEAM-maintainers gdi-TEAM-approvers
```

I'm pretty sure we need to specify the full team identifier (including the org) for them to be recognized by GitHub. Also, we should prevent privileged escalations by restricting access to the `CODEOWNERS` file.

```suggestion
* @signalfx/gdi-TEAM-maintainers @signalfx/gdi-TEAM-approvers

CODEOWNERS @signalfx/gdi-TEAM-maintainers
```